operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-service-monitor.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chains-metrics-reader
+  namespace: openshift-pipelines
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: chains-metrics-reader
+  namespace: openshift-pipelines
+  annotations:
+    kubernetes.io/service-account.name: chains-metrics-reader
+    argocd.argoproj.io/sync-wave: "1"
+type: kubernetes.io/service-account-token
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: chains-service-metrics-reader
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+rules:
+  - nonResourceURLs:
+      - /metrics
+    verbs:
+      - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus-chains-service-metrics-reader
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: chains-service-metrics-reader
+subjects:
+  - kind: ServiceAccount
+    name: chains-metrics-reader
+    namespace: openshift-pipelines
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: tekton-chains-controller
+  namespace: openshift-pipelines
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+spec:
+  endpoints:
+    - path: /metrics
+      port: metrics
+      scheme: http
+      bearerTokenSecret:
```

Comment:
ok lengthy discussion with @avinal @sayan-biswas and myself in https://redhat-internal.slack.com/archives/C032EJ007C0/p1692878986840629

we are losing this use of bearer token in service monitor, and the secret it points to